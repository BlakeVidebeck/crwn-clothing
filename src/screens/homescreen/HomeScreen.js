import React from 'react'

import Directory from '../../components/directory/Directory'

import { HomeScreenContainer } from './HomeScreenStyles'

const HomeScreen = () => {
	return (
		<HomeScreenContainer>
			<Directory />
		</HomeScreenContainer>
	)
}

export default HomeScreen
