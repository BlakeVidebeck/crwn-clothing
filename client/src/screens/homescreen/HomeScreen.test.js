import React from 'react'
import { shallow } from 'enzyme'
import HomeScreen from './HomeScreen'

it('should render HomeScreen component', () => {
	expect(shallow(<HomeScreen />)).toMatchSnapshot()
})
