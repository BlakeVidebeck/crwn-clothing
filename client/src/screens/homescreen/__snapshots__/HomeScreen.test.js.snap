// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomeScreen component 1`] = `
<styled.div>
  <Connect(Directory) />
</styled.div>
`;
